web-app/public/locales/en/gbfs.json

```
@@ -0,0 +1,24 @@
+{
+    "versions": "Versions",
+    "autodiscoveryUrl": "Auto-Discovery Url",
+    "autodiscoveryUrlCopied": "Auto-Discovery Url Copied",
+    "runValidationReport": "Run Validation Report",
+    "openFeedUrl": "Open Feed URL",
+    "features": {
+        "gbfs": "GBFS",
+        "system_regions": "System Regions",
+        "system_information": "System Information",
+        "geofencing_zones": "Geofencing Zones",
+        "system_pricing_plans": "System Pricing Plans",
+        "vehicle_types": "Vehicle Types",
+        "vehicle_status": "Vehicle Status",
+        "station_status": "Station Status",
+        "system_calendar": "System Calendar",
+        "system_hours": "System Hours",
+        "gbfs_versions": "GBFS Versions",
+        "station_information": "Station Information",
+        "system_alerts": "System Alerts",
+        "free_bike_status": "Free Bike Status",
+        "nearby_stations": "Nearby Stations"
```

These endpoints should be removed as they are not "features" (as per https://github.com/MobilityData/mobility-feed-api/issues/1125): 
`gbfs`, `system_information`, `system_calendar`, `system_hours` and `nearby_stations`. Thanks!

```suggestion
        "system_regions": "System Regions",
        "geofencing_zones": "Geofencing Zones",
        "system_pricing_plans": "System Pricing Plans",
        "vehicle_types": "Vehicle Types",
        "vehicle_status": "Vehicle Status",
        "station_status": "Station Status",
        "gbfs_versions": "GBFS Versions",
        "station_information": "Station Information",
        "system_alerts": "System Alerts",
        "free_bike_status": "Free Bike Status",
```

---

Hey @richfab! Yes - the features fix will happen in #1125 - it's out of scope for this issue

---

I understand the backend fix will happen in https://github.com/MobilityData/mobility-feed-api/issues/1125.
The UI code shouldn't need logic for non-feature endpoints, unless I'm mistaken.